doc/lsst.cp.pipe/constructing-calibrations.rst

```diff
@@ -0,0 +1,733 @@
+        u/czw/DM-28920/tempPtcA.0721 \
+        --begin-date 2019-01-01 --end-date 2050-01-01 ptc
+    butler certify-calibrations /repo/main \
+        u/czw/DM-28920/ptcGen.20210721b \
+        u/czw/DM-28920/tempPtcB.0721 \
+        --begin-date 2019-01-01 --end-date 2050-01-01 ptc
+
+- That PTC can be used to construct a linearity solution:
+.. code:: bash
+
+    butler certify-calibrations /repo/main \
+        u/czw/DM-28920/linearityGen.20210721Xa \
+        u/czw/DM-28920/calib/linearity.20210721 \
+        --begin-date 2020-01-01 --end-date 2050-01-01 linearity
+    butler collection-chain /repo/main --mode=extend \
+        u/czw/DM-28920/calib.20210720 \
+        u/czw/DM-28920/calib/linearity.20210721
+
+- Which can be used to update the PTC and remove linearity effects:
+.. code:: bash
+
+    butler certify-calibrations /repo/main\
+        u/czw/DM-28920/ptcGen.20210721Ya \
+        u/czw/DM-28920/ptcA.20210721 \
+        --begin-date 2019-01-01 --end-date 2050-01-01 ptc
+    butler certify-calibrations /repo/main
+        u/czw/DM-28920/ptcGen.20210721Yb \
+        u/czw/DM-28920/ptcB.20210721 \
+        --begin-date 2019-01-01 --end-date 2050-01-01 ptc
+
+- The updated PTC can be used to create a brighter-fatter kernel:
+.. code:: bash
+
+    butler certify-calibrations /repo/main \
+        u/czw/DM-28920/bfkGen.20210721a \
+        u/czw/DM-28920/bfk.20210721 \
+        --begin-date 2020-01-01 --end-date 2050-01-01 bfk
+    butler collection-chain /repo/main --mode=extend \
+        u/czw/DM-28920/calib.20210720 \
+        u/czw/DM-28920/calib/bfk.20210721
+
+- With a full set of calibrations, the crosstalk can be measured:
+.. code:: bash
+
+   butler certify-calibrations /repo/main \
+       u/czw/DM-28920/crosstalkGen.20210721a \
+       u/czw/DM-28920/crosstalk.20210721
+       --begin-date 2020-01-01 --end-date 2050-01-01 crosstalk
+   butler collection-chain /repo/main --mode=extend \
+       u/czw/DM-28920/calib.20210720 u/czw/DM-28920/calib/crosstalk.20210721
+
+.. _cp-pipe-readNoise:
+
+Read Noise
+==========
+
+Calibration construction and verification are sensitive to the read noise value listed in the ``camera`` camera geometry definition.  Inaccurate values may trigger test failures that are spurious.  Setting the ``isr:doEmpiricalReadNoise=True`` option during the bias processing (as the bias generally has very little signal other than noise) may be necessary to bootstrap a full set of calibrations from scratch.  This option records the values measured in the log, and by analyzing the results of many exposures, better estimates of the read noise can be generated.
+
+
+.. _cp-pipe-biases:
+
+Constructing biases
+===================
+
+- Identify a set of exposures to use as inputs from the repository:
+.. code:: bash
+
+    butler query-dimension-records /repo/main exposure \
+        --where "instrument='LATISS' AND exposure.observation_type='bias' \
+                 AND exposure.target_name='Park position' \
```

Comment:
I'm a bit worried that the use of "Park position" here is working around a header problem that needs fixing. What does it mean for a bias observation_type to have a target_name other than "Park position"?

Reply:
This set of WHERE options was what I came to after looking through the unrestricted results.
I believe `target_name` can be anything.  Other example values are "UNKNOWN" (excluded for being uninformative), "FlatField position" (excluded to prevent the possibility the flat screen is being illuminated, potentially creating light leaks), "HD 20894" (is a star), "azel_target" (excluded as it seemed the minority compared to "Park position"; eliminates light leak differences), "park" (same as before).  Basically, based on the rough counts, I took "Park position" to be biases taken by a bias script or procedure, and the remainder being manual tests that I didn't want to touch.
The dark_time constraint excludes exposures in the upper ~10th percentile of dark_times.  I don't know how those exposures were taken, but I do not want them.  There probably should be a lower constraint as well, bounding the expected dark time for a bias (~0.095 based on the median).